Client/src/Components/TerryComponent/reusable/AlertMessage.jsx

```diff
@@ -0,0 +1,74 @@
+// eslint-disable-next-line no-unused-vars
+import React from "react";
+import "./AlertMessage.css";
+import PropTypes from "prop-types";
+import { Snackbar, Alert } from "@mui/material";
+
+const AlertMessage = ({
+  open,
+  type,
+  title = "",
+  message,
+  dismissText = "Learn more",
+  actionText,
+  close,
+  dismissClose,
+}) => {
+  // Function to determine icon color based on the title
+  const switchIconColor = (type) => {
+    switch (type) {
+      case "release":
+        return "black";
+      case "problem":
+        return "red";
+      case "update":
+        return "green";
+      default:
+        return "blue";
+    }
+  };
+
+  return (
+    <Snackbar open={open} className="snackbar-main">
+      <Alert
+        onClose={close}
+        severity="info"
+        variant="filled"
+        sx={{
+          width: "100%",
+          backgroundColor: "white",
+          color: "black",
+          border: "2px solid lightgrey",
+          "& .MuiAlert-icon": {
```

Comment:
This is a sample component so it's not a big deal, but if this were a production component we would not want to hardcode styles.

Styles are ideally built off our MUI theme if they are used across components, or in the component's CSS file if it is a one off component